### PR TITLE
Force quit and subscription page fixes

### LINKED ISSUE #161: Subscription page breaks when trying to sort by hot/new/etc.
Traceback (most recent call last):
  File "/usr/bin/rtv", line 9, in <module>
    load_entry_point('rtv==1.6', 'console_scripts', 'rtv')()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 558, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2682, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2355, in load
    return self.resolve()
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2361, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/__main__.py", line 91, in <module>
    sys.exit(main())
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/__main__.py", line 78, in main
    page.loop()
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/subreddit.py", line 51, in loop
    self.controller.trigger(cmd)
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/page.py", line 225, in trigger
    return func(self.instance, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/subreddit.py", line 185, in open_subscriptions
    page.loop()
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/subscription.py", line 34, in loop
    self.controller.trigger(cmd)
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/page.py", line 225, in trigger
    return func(self.instance, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/rtv-1.6-py3.5.egg/rtv/page.py", line 288, in sort_content_hot
    self.refresh_content(order='hot')
TypeError: refresh_content() got an unexpected keyword argument 'order'

What would be the most elegant bypass? As a quick hack, I did

```
@SubscriptionController.register('1', '2', '3', '4', '5')
def ignore_sorting(self):
    pass (or curses.flash, or show_notification but that one acts strangely)
```

on `subscription.py ` but I can see why that wouldn't be the proper solution. Maybe importing that class to `page.py` and adding cases to each sorting controller? Something like this:

```
@BaseController.register('1')
def sort_content_hot(self):
    if not isinstance(page, SubscriptionPage):
        self.refresh_content(order='hot')
```

I prefer to treat this on the subclass since that's its responsibility, and since it shouldn't do anything when those keys are pressed, maybe the `pass` wouldn't be as ugly as it seems to be.

### PR DESCRIPTION
**Pull request by zambonin**
As asked by @shaggytwodope on #156, and fixes #161.

### CODE REVIEW

**Comment by shaggytwodope** [comment]
Brillant, thanks for your work. RTV getting better everyday.

File: rtv/docs.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
   `r` or `F5`         : Refresh page content
   `u`                 : Log in or switch accounts
   `?`                 : Show the help screen
-  `q`                 : Quit
+  `q/Q`               : Quit/Force quit
```
Comment (michael-lazar) [inline rtv/docs.py]: We should add this to the README too.

Comment (zambonin) [inline rtv/docs.py]: added. :) also fixed #163 if i'm not mistaken.

**Comment by michael-lazar** [comment]
Looks great, thanks!

**Pull request merged**